### PR TITLE
Adding Google Gemini prompting support, along with some other fixes

### PR DESCRIPTION
**Pull request by leomarusic**
- Added Google Gemini text prompting
- Fixed calendar working on Windows/Linux systems
- Calendar directory is now being created if it does not exist
- Removed the redundant email confirmation from the user registration process
- User will now automatically login after registration
- User can now set an username (had to make a custom sign-in manager for that… wonderful)
- Also updated .gitignore to include user data files

### CODE REVIEW

File: IzvorniKod/wwwroot/css/chat.css [hunk 1]
```diff
@@ -0,0 +1 @@
+﻿
```
Comment (filipbelina) [inline IzvorniKod/wwwroot/css/chat.css:1]: Jeli ovo samo prazan css file, ako je jeli potreban pa je tu ili neki drugi razlog?

Comment (leomarusic) [reply IzvorniKod/wwwroot/css/chat.css:1]: Taj file je tu da ga frontend tim može editati kada uređivaju tu stranicu.

Comment (filipbelina) [reply IzvorniKod/wwwroot/css/chat.css:1]: Thinking about the future that's what makes you a good emply,
okej np resolved

File: IzvorniKod/Views/Home/Chat.cshtml.cs
Comment (jakovlovakovic) [inline IzvorniKod/Views/Home/Chat.cshtml.cs:1]: Je li ovaj file potreban ako renderamo Chat view pomoću kontrolera?

**Review by jakovlovakovic — APPROVED** [review]
Pregledan pull request. Fokus na backend dijelu. ChatController i ChatService rađeni po dogovorenom standardu. Izmjeni u ostalim kontrolerima su opravdane. Service ispravno registriran. Također smatram da su ostali fileovi uredu. Approved.

File: IzvorniKod/Views/Home/Chat.cshtml.cs
Comment (leomarusic) [reply IzvorniKod/Views/Home/Chat.cshtml.cs:1]: Nije potreban, ali je tu za buduće feature.

**Pull request merged**